include/rcpputils/filesystem_helper.hpp

```diff
@@ -41,6 +41,7 @@
 
 #include <string>
 #include <vector>
+#include <cstdint>
```

Comment:
do you mind to include `<ostream>`, it used in the `operator<<` but it's not included

Reply:
I have included the requested header

Reply:
do you mind to alphabetize the heraders? then it's good to go with green CI